.mailmap

```diff
@@ -22,15 +23,20 @@ Hui Kong <hui.kong@qunar.com>
 Ian Vernon <ian@cilium.io> <vagrant@k8s1>
 Ifeanyi Ubah <ify1992@yahoo.com>
 Jarno Rajahalme <jarno@covalent.io>
+Jed Salazar <jed@isovalent.com>
 Jerry J. Muzsik <jerrymuzsik@icloud.com>
 Jomen Xiao <jomenxiao@gmail.com>
+Joshua Roppo <joshroppo@gmail.com>
 Junli Ou <oujunli306@gmail.com>
 Liu Qun <qunliu@zyhx-group.com>
 Madhu Challa <challa@gmail.com>
 Matthew Gumport <me@gum.pt>
 Michael Vorburger <vorburger@redhat.com>
 Peiqi Shi <uestc.shi@gmail.com>
-Sergey Generalov <sergey@genbit.ru>
+Sean Winn <sean@isovalent.com> <seanmwinn@hotmail.com>
+Sergey Generalov <sergey@isovalent.com> <sergey@genbit.ru>
 Thomas Graf <thomas@cilium.io>
+Tony Lu <tonylu@linux.alibaba.com> <tonylu@linux.alibaba.com>
```

Comment:
There's some bug fix the script that lists those emails?

Reply:
Yes, will send a separate PR. It'll be obvious when you see it :-)